src/util/hass-util.html

```diff
@@ -218,21 +232,17 @@
     case 'occupancy':
       return activated ? 'mdi:home' : 'mdi:home-outline';
     case 'opening':
-      return activated ? 'mdi:door-closed' : 'mdi:door-open';
-    case 'sound':
-      return activated ? 'mdi:music-note-off' : 'mdi:music-note';
-    case 'vibration':
-      return activated ? 'mdi:crop-portrait' : 'mdi:vibrate';
-    case 'gas':
-    case 'power':
-    case 'problem':
-    case 'safety':
-    case 'smoke':
-      return activated ? 'mdi:verified' : 'mdi:alert';
+      return activated ? 'mdi:square-outline' : 'mdi:square';
```

Comment:
Looks like these are backwards. (Note that `activated` is actually incorrectly named. Something that needs to be cleaned up still.)

Reply:
Doh, good catch...

New commit pushed.